### ISSUE TITLE
Alternative support for Google Speech to Text (feature request)

### ISSUE THREAD

**Issue by slesinger** [stanford-oval/almond-server#36]
For people from countries who speak languages not supported by Microsoft speech it would be a great feature to use Google STT and TTS.

**Comment by gcampax** [comment]
Hi @slesinger! Thank you for your suggestion. We're not planning to integrate with other backends ourselves, but would be more than willing to review and merge patches that do so. For those backends, users would also be responsible for acquiring the necessary API keys.

**Comment by prologic** [comment]
What about self-hosted speec recognition?

**Comment by gcampax** [comment]
Are you referring to deepspeech? It is on our roadmap to support it, but currently has very low priority.

**Comment by prologic** [comment]
Yes something like this. I've actually played with it but unfortunately the setup is quite complicated -- And without a GPU its rather slow :/
